NuKeeper.Tests/Engine/GitHubEngineTests.cs

```diff
@@ -1,13 +1,14 @@
-using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using NSubstitute;
 using NuKeeper.Abstractions.Configuration;
+using NuKeeper.Abstractions.DTOs;
 using NuKeeper.Abstractions.Logging;
 using NuKeeper.Engine;
-using NuKeeper.GitHub;
 using NuKeeper.Inspection.Files;
 using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NuKeeper.Abstractions.CollaborationPlatform;
```

Comment:
Is using sorting disabled?

Reply:
looks like when using resharper to refactor it doesn't re-order afterwards... will go through and fix that now